gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -266,6 +277,11 @@ static void addJobStatusToStateStore(org.apache.gobblin.configuration.State jobS
 
       modifyStateIfRetryRequired(jobStatus);
       stateStore.put(storeName, tableName, jobStatus);
+
+      if (isStateTransitionToFinal(jobStatus, states) && eventProducer.isPresent()) {
+        log.info("I am here");
```

Comment:
Did you mean to leave this here 😆

Reply:
Oops I was debugging something and it got caught 😂